components/PoeModal.tsx

```diff
@@ -643,6 +689,22 @@ const PoeModal = () => {
 
                             setTimeoutAndHash(timeoutAndHash);
 
+                            if (signImmediatelyFlag) {
+                              const storage = toStorage(
+                                versioned.version,
+                                await cc.storage(),
+                                BigNumber(0)
+                              );
+                              const proposal_id = storage.proposal_counter;
+                              timeoutAndHash = await versioned.signProposal(
+                                cc,
+                                state.connection,
+                                proposal_id,
```

Comment:
```suggestion
                              const proposalId = storage.proposal_counter;
                              timeoutAndHash = await versioned.signProposal(
                                cc,
                                state.connection,
                                proposalId,
```

Only some nitpicking, but make sure to use camelCase instead of snake_case

Reply:
Done